js/templates/userPage/userPage.html

```diff
@@ -48,7 +48,7 @@ <h1 class="txBg txUnb txUnl clrTGlow js-tabTitle">
       </h1>
     </div>
     <div>
-      <div class="btnStrip floR clrSh2">
+      <div class="btnStrip floR clrSh2 margTLg">
```

Comment:
A better way to align this to the bottom would be to add a flex class that has align-self: flex-end. I'll add a patch. There are probably several cases where we'll want something like that.